ui/app/components/select-dropdown.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@ember/component';
+
+/**
+ * @module SelectDropdown
+ * SelectDropdown components are used to render a dropdown.
+ *
+ * @example
+ * ```js
+ * <SelectDropdown dropdownLabel='Date Range' @options={{options}} @onChange={{onChange}}/>
+ * ```
+ *
+ * @param dropdownLabel=null {String} - The label for the select element.
```

Comment:
Could this be just `label`? (sorry very much a 🚲 🏚 )

Reply:
certainly!